components/home/RecentTeaching/RecentTeaching.tsx

```diff
@@ -128,7 +128,10 @@ export default function RecentTeaching(): JSX.Element {
             <View style={style.container}>
                 {teachingImage?.url ?
                     <View style={{ width: '100%' }}>
-                        <TouchableWithoutFeedback onPress={() => navigation.navigate('SermonLandingScreen', { item: teaching })} >
+                        <TouchableWithoutFeedback onPress={() => {
```

Comment:
Why was this behaviour changed?

Reply:
I must have added it while testing. I don't think this is needed.

Reply:
I just remembered that I encountered a bug and that is why I added this.
When navigating from RecentTeaching -> SermonsLandingScreen ->  then clicking on the series title it would error out because SeriesLandingScreen is in TeachingStack.